scripts/lib/api/processHtml.ts

```diff
@@ -70,30 +70,25 @@ export function loadImages(
   imageDestination: string,
   isReleaseNotes: boolean,
 ): Image[] {
-  const images: Image[] = [];
-  $main
+  return $main
     .find("img")
     .toArray()
-    .forEach((img) => {
+    .map((img) => {
       const $img = $(img);
 
       const imageUrl = new URL($img.attr("src")!, url);
       const src = imageUrl.toString();
 
       const filename = last(src.split("/"));
-      const dest = `${imageDestination}/${filename}`;
-
-      $img.attr("src", dest);
-
+      let dest = `${imageDestination}/${filename}`;
       if (isReleaseNotes) {
         // Release notes links should point to the current version
-        $img.attr("src", dest.replace(/[0-9].*\//, ""));
+        dest = dest.replace(/[0-9].*\//, "");
       }
```

Comment:
This is a change. We before returned the wrong `dest` in the `Image` type and we were only fixing the `dest` for the HTML itself.

Reply:
Good catch! This change is important to have in mind when we regenerate the API docs in the future because we will override the images of the current version, but I don't see why not with the new release notes mechanism.